### ISSUE TITLE
Function for DNN 9+

### ISSUE THREAD

**Issue by in8dna** [sachatrauwaen/SQL_OpenUrlRewriter#1]
I have many dnn installations that need upgrading to 9+. Can we discuss costs for making this module work in version 9?

**Comment by in8dna** [comment]
ping on email receipt?

**Comment by sachatrauwaen** [comment]
Hi, yes if you sponcor the development. I can do it.

**Comment by in8dna** [comment]
Yes, I will sponsor. Should we email arrangements and details?

**Comment by in8dna** [comment]
I left my contact info on your website contact form.

**Comment by in8dna** [comment]
here's my contact form for you to leave your email...
https://scotomaville.com/contact

**Comment by in8dna** [comment]
yes to your email questions. We will sponsor, and assist in testing. My email replies to you are blocked as spam by gmail. Arrgg.

**Comment by in8dna** [comment]
This offer is still good. The need is now a priority. How's your schedule to complete this by the end of This month?
